walk-in-interview/src/main/webapp/account/create-account/business/script.js

```diff
@@ -0,0 +1,222 @@
+ */
+function onLogIn() {
+  // TODO(issue/101): Display button according to log in status;
+
+  // TODO(issue/100): set the cookie at the server side instead
+  setCookie(USER_TYPE_COOKIE_PARAM, USER_TYPE_BUSINESS);
+
+  // TODO(issue/102): replace with proper notification
+  alert(STRINGS['new-user-info']);
+}
+
+function onLogOut() {
+  // TODO(issue/101): Display button according to log in status;
+
+  // TODO(issue/102): replace with proper notification
+  alert(AUTH_STRINGS['sign-out-success']);
+}
+
+function onLogInFailure() {
+  // TODO(issue/101): Display button according to log in status;
+  
+  // TODO(issue/102): replace with proper notification
+  alert(ACCOUNT_STRINGS['create-account-error-message']);
+}
+
+
+function onLogOutFailure() {
+  // TODO(issue/101): Display button according to log in status;
+}
+
+/** Adds all the text to the fields on this page. */
+function renderPageElements() {
+  const accountElement = document.getElementById('account');
+
+  const accountLabelElement = accountElement.children[0];
+  accountLabelElement.innerText = STRINGS['account'];
+
+  const accountInputElement = accountElement.children[1];
+  accountInputElement.setAttribute('name', 'account-input');
+  accountInputElement.setAttribute('type', 'text');
+
+  const passwordElement = document.getElementById('password');
+
+  const passwordLabelElement = passwordElement.children[0];
+  passwordLabelElement.innerText = STRINGS['password'];
+
+  const passwordInputElement = passwordElement.children[1];
+  passwordInputElement.setAttribute('name', 'password-input');
+  passwordInputElement.setAttribute('type', 'password');
+
+  const backButton = document.getElementById('back');
+  backButton.innerText = ACCOUNT_STRINGS['back'];
+
+  const submitButton = document.getElementById('submit');
+  submitButton.innerText = ACCOUNT_STRINGS['submit'];
+}
+
+const backButton = document.getElementById('back');
+backButton.addEventListener('click', (_) => {
+  window.location.href = CREATE_ACCOUNT_HOMEPAGE_PATH;
+});
+
+const submitButton = document.getElementById('submit');
+submitButton.addEventListener('click', async (_) => {
+  // Disables the button to avoid accidental double click
+  document.getElementById('submit').disabled = true;
+
+  const account = document.getElementById('account-input').value.trim();
+
+  if (account === '') {
+    setErrorMessage(/* errorMessageElementId= */'error-message',
+        /* msg= */ STRINGS['empty-email'],
+        /* includesDefault= */false);
+    return;
+  }
+
+  const password = document.getElementById('password-input').value.trim();
+
+  // Resets the error (there might have been an error msg from earlier)
+  setErrorMessage(/* errorMessageElementId= */'error-message',
+      /* msg= */ '',
+      /* includesDefault= */false);
+
+  await Auth.createBusinessAccount(account, password)
+      .then(async() => {
+        await createEmptyAccount();
```

Comment:
nit: the name of the method is a bit weird. when reading i wonder why you have createBusinessAccount followed by createEmptyAccount. can you find a better name?